src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -548,6 +548,15 @@ export class Connector implements IConnector {
                         await this.getFavicon((await this._dom.querySelectorAll('link[rel~="icon"]'))[0]);
                     }
 
+                    await new Promise((resolve) => {
+                        Tracing.tracingComplete(() => {
+                            this._server.emitAsync('tracing::end', event);
+                            resolve();
+
+                        });
+                        debug(`Ending Collecting Chrome traces`);
+                        Tracing.end();
```

Comment:
Can `Tracing.end` throw an exception? if so, you can wrap it in a `try/catch` and call to `reject` in the `catch`